src/lib/salad/hll_empirical.c

```diff
@@ -0,0 +1,5362 @@
+#include "hll_empirical.h"
+
+#include <trivia/util.h>
+
+#define DATA_POINTS  200
+
+/*
+ * Estimation data that is used to determine the bias correction.
+ * For example, if raw estimation value lies on segment from
+ * estimation_data[p][idx] to estimation_data[p][idx+1] the
+ * corresponding bias correction lies on segment from
+ * estimation_bias_data[p][idx] to estimation_bias_data [p][idx+1]
+ * and can be interpolated. For more details see http://goo.gl/iU8Ig.
+ */
+static const double estimation_data[][DATA_POINTS] = {
```

Comment:
This is the data from the article, and it is under a license prohibiting commercial use.
Instead of point-by-point interpolation, I generated bias correction curves on my own. I can share a script that generates curves, but I do not know where to put it.

Reply:
Added scripts with some instructions. See tools/.